nixos/modules/services/x11/window-managers/i3.nix

```diff
@@ -52,13 +52,12 @@ in
       default = with pkgs; [
         dmenu
         i3status
-        i3lock
       ];
       defaultText = literalExpression ''
         with pkgs; [
           dmenu
           i3status
-          i3lock
+          # i3lock also gets enabled by default via programs.i3lock.enable
```

Comment:
We should remove that line entirely as it is misleading

Reply:
done